pkg/labels/labels.go

```diff
@@ -90,6 +89,9 @@ const (
 	// PIDContainer is the `nerdctl run --pid` for restarting
 	PIDContainer = Prefix + "pid-container"
 
+	// RunError encapsulates a container run error.
+	RunError = Prefix + "run-error"
```

Comment:
Wonder if this should be `start-error` or just `error` or `errors`

Reply:
`start-error` LGTM. `error` has a large scope.

Reply:
switch for `error` to handle `stop` and `remove` verb(s)